cpp/11501_stock.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+
+long long stock(const int cnt);
+
+int main() {
+
+    int N;
+
+    std::cin >> N;
+
+    long long* result = new long long[N];
+
+
+    int cnt;
+    for (int i = 0; i < N; i++) {
+        std::cin >> cnt;
+        result[i] = stock(cnt);
+    }
+
+    for (int i = 0; i < N; i++) {
+        std::cout << result[i] << '\n';
+    }
```

Comment:
Range-based for (C++11)를 사용하면:
```suggestion
    for (const long long& it: result) std::cout << it << '\n';
```